core/biogeochem/POPLUC.F90

```diff
@@ -1439,18 +1458,20 @@ SUBROUTINE POP_LUC_CASA_transfer(POPLUC, POP, LUC_EXPT, &
              irp = ilu + j -1
              ! Update tile area
              if (ilu == p) then
-                POPLUC%primf(g)  = max(patch(irp)%frac + dA_r(ilu) + dA_d(ilu), 0.0_dp)
+                POPLUC%primf(g) = min(1.0, max(patch(irp)%frac + dA_r(ilu) + dA_d(ilu), 0.0_dp))
```

Comment:
My slight concern here is that if we need to constrain things between 0 and 1 here then we (likely) need a compensatory  adjustment to one of `dA_r(ilu)` and/or `dA_d(ilu)` which then feeds onto the LUC states.

Reply:
Good point. I probably haven't thought this through. I will keep it in mind for the next commit. For now I will double-check the patchfracs in the results and ensure they all add up to 1 (which I am pretty sure they do).